shared/provision/gpg-sign/index.desktop.js

```diff
@@ -12,43 +12,63 @@ class GPGSign extends Component<Props> {
     return (
       <Container style={styles.container} onBack={() => this.props.onBack()}>
         <RowCSS />
+        {this.props.importError && (
```

Comment:
can you explain what happens in this flow. Does this error happen immediately before this screen, or we see this screen, then they click this top button , then they get this error? If so would it be simpler to just keep the original screen and just put an error below the button  (essentially just plumbing through the error) and not do any other kind of state or switching. they can click again if they fixed it, or they'll get the same error again, or they can choose the second option. then the 2 different paths for the mergeProps in the container can go away